build.csx

```diff
@@ -0,0 +1,217 @@
+#load "packages/simple-targets-csx.5.2.0/simple-targets.csx"
+
+#r "System.Xml.Linq"
+
+//using System.Runtime.Serialization.Json;
+//using System.Xml;
+//using System.Xml.Linq;
+using static SimpleTargets;
+
+static readonly var  solutionName = "FakeItEasy";
+
+static readonly var solution = "./" + solutionName + ".sln";
+static readonly var packagesDir = Path.GetFullPath("packages");
+static readonly var repoUrl = "https://github.com/FakeItEasy/FakeItEasy";
+static readonly var versionAssembly = "./src/FakeItEasy/bin/Release/FakeItEasy.dll";
+static readonly IEnumerable<string> nuspecs = Directory.GetFiles("./src", "*.nuspec");

```

Comment:
I'd done so to highlight that they shouldn't be modified. `string[]`s could have their innards replaced more easily. @thomaslevesque had been pushing for `const` variables, so I thought it was a concern. I'm content with anything, even non-readonly vars.
So, what do we want?

* completely loosey-goosey variables and we just trust the script not to muck about with them,
* readonly variables and we trust the script not to muck about with array innards, or
* locked-down as much as we can variables?